### PR TITLE
Improve rit init command

### PR DESCRIPTION
**Pull request by kaduartur**
### Description
Improve `rit init`command and add the default language to English.

### How to verify it
It's a small part of the #775 issue, a separate pull request was created to facilitate code review

### CODE REVIEW

File: pkg/cmd/init.go [hunk 1]
```diff
@@ -119,19 +123,35 @@ func NewInitCmd(
...
 		}
 
-		if err := in.setRunnerType(); err != nil {
+		runType, err := in.setRunnerType()
+		if err != nil {
 			return err
 		}
 
-		prompt.Success("\nInitialization successful!\n")
+		prompt.Success("\n✅  Initialization successful!\n")
```
Comment (brunasilvazup) [inline pkg/cmd/init.go:143]: We had problems with `✅` on certain OSs, which I do not remember which ones, did we come back with them?

Comment (kaduartur) [reply pkg/cmd/init.go:143]: We can send it to QA and tests it in Linux and Windows. It works in macOS.

File: pkg/cmd/init.go [hunk 2]
```diff
@@ -358,3 +376,23 @@ func (in initCmd) tutorialInit() error {
 
 	return nil
 }
+
+func (in initCmd) Welcome() {
+	const welcome = " _______       _     _             __         _                           ______    _____      _____  \n|_   __ \\     (_)   / |_          [  |       (_)                        .' ___  |  |_   _|    |_   _| \n  | |__) |    __   `| |-'  .---.   | |--.    __    .---.     ______    / .'   \\_|    | |        | |   \n  |  __ /    [  |   | |   / /'`\\]  | .-. |  [  |  / /__\\\\   |______|   | |           | |   _    | |   \n _| |  \\ \\_   | |   | |,  | \\__.   | | | |   | |  | \\__.,              \\ `.___.'\\   _| |__/ |  _| |_  \n|____| |___| [___]  \\__/  '.___.' [___]|__] [___]  '.__.'               `.____ .'  |________| |_____| \n\n"
```
Comment (brunasilvazup) [inline pkg/cmd/init.go:381]: https://media.giphy.com/media/BoItb0QoZ8G2nuqGQb/giphy.gif

Comment (brunasilvazup) [inline pkg/cmd/init.go:381]: image

File: pkg/cmd/upgrade_test.go [hunk 3]
```diff
@@ -70,7 +70,7 @@ func TestUpgradeCmd_runFunc(t *testing.T) {
 		resolver  version.Resolver
 		Manager   upgrade.Manager
 		UrlFinder upgrade.UrlFinder
-		input     prompt.InputList
+		input     prompt.InputBool
 		file      stream.FileWriteReadExister
 		tag       git.Tag
 	}
```
Comment (henriquemoraeszup) [inline pkg/cmd/upgrade_test.go:76]: How about a refactor for the new test format?

**Review by henriquemoraeszup — COMMENTED** [review]
A couple of questions:
* I remember emojis were not working on windows, is this case fine?
* More important than registering the default English, is the current version statically setting the English language. We are not selecting the language based on the config yet right?

File: pkg/cmd/init.go [hunk 4]
```diff
@@ -256,50 +270,53 @@ You can view our Privacy Policy (http://insights.zup.com.br/politica-privacidade
...
 
-	return nil
+	return responseToWrite, nil
 }
 
-func (in initCmd) setRunnerType() error {
-	selected, err := in.List(SelectFormulaTypeQuestion, formula.RunnerTypes)
+func (in initCmd) setRunnerType() (formula.RunnerType, error) {
+	prompt.Info("🏃 FORMULA RUN TYPE 🏃\n")
+	runTypes := []string{"🏠 local", "🐳 docker"}
```
Comment (brunasilvazup) [inline pkg/cmd/init.go:281]: This runTypes could be placed as const in the beginning file? - to use later in the tests -

**Review by brunasilvazup — COMMENTED** [review]
So far, it's a good refactoring and improvement in init, I will wait for the changes in the tests to finalize the review.

- Could you put a gif about the init process in the description, please?

**Comment by henriquemoraeszup** [comment]
> A couple of questions:
> 
> ...
> * More important than registering the default English, is the current version statically using the English language. We are not selecting the language based on the config yet right?

**Comment by kaduartur** [comment]
> A couple of questions:
> 
> * I remember emojis were not working on windows, is this case fine?
> * More important than registering the default English is the current version statically using the English language. We are not selecting the language based on the config yet, right?

We are not selecting the language based on the config, we will do that in the next PR!

File: pkg/cmd/upgrade_test.go (hunk 3 above)
Comment (henriquemoraeszup) [reply pkg/cmd/upgrade_test.go:76]: bump

**Comment by henriquemoraeszup** [comment]
I am still missing the test comment reply

File: pkg/cmd/upgrade_test.go (hunk 3 above)
Comment (kaduartur) [reply pkg/cmd/upgrade_test.go:76]: I think it makes sense to do in another PR

File: internal/mocks/mocks.go [hunk 5]
```diff
@@ -381,3 +382,31 @@ func (w *WorkspaceMock) List() (formula.Workspaces, error) {
...
+
+func (r *RitConfigMock) Read() (config.Configs, error) {
+	args := r.Called()
+	return args.Get(0).(config.Configs), args.Error(1)
+}
+
+func (r *RitConfigMock) Write(configs config.Configs) error {
+	args := r.Called(configs)
+	return args.Error(0)
+}
```
Comment (henriquemoraeszup) [inline internal/mocks/mocks.go:412]: Can you add this to a separate file with the package name?

File: pkg/cmd/init_test.go [hunk 6]
```diff
@@ -22,366 +22,273 @@ import (
...
-				},
-				tutorial: TutorialFinderMock{},
-				inBool:   inputTrueMock{},
+			name: "error to write metric file",
+			in: in{
+				gitTag:         git.Tag{Name: "1.0.0"},
+				tutorialHolder: rtutorial.TutorialHolder{Current: rtutorial.DefaultTutorial},
+				inBool:         true,
+				fileWriteErr:   errors.New("error to write metric file"),
 				inList: inputListCustomMock{
```
Comment (henriquemoraeszup) [inline pkg/cmd/init_test.go:184]: How about the list mock?

File: pkg/cmd/init_test.go [hunk 7]
```diff
@@ -22,366 +22,273 @@ import (
...
+				tutorialMock,
+				configRunnerMock,
+				fileMock,
+				in.inList,
+				inBoolMock,
 				metricSender,
+				ritConfigMock,
 			)
 
 			initPrompt.PersistentFlags().Bool("stdin", false, "input by stdin")
```
Comment (henriquemoraeszup) [inline pkg/cmd/init_test.go:294]: We should support flags for the init command

Comment (kaduartur) [reply pkg/cmd/init_test.go:294]: I think this is good, but in order not to escape further from the context of this PR I prefer to do it in a separate task

File: pkg/cmd/init_test.go (hunk 6 above)
Comment (kaduartur) [reply pkg/cmd/init_test.go:184]: Do you know any alternative to custom mocks?

Comment (kaduartur) [reply pkg/cmd/init_test.go:184]: I will fix 🚒

**Review by brunasilvazup — APPROVED** [review]

File: pkg/cmd/init_test.go [hunk 8]
```diff
@@ -22,366 +22,233 @@ import (
...
+	"github.com/ZupIT/ritchie-cli/internal/mocks"
 	"github.com/ZupIT/ritchie-cli/pkg/git"
 	"github.com/ZupIT/ritchie-cli/pkg/metric"
-	"github.com/ZupIT/ritchie-cli/pkg/prompt"
 	"github.com/ZupIT/ritchie-cli/pkg/rtutorial"
-	"github.com/ZupIT/ritchie-cli/pkg/stream"
-	sMocks "github.com/ZupIT/ritchie-cli/pkg/stream/mocks"
 )
 
 func Test_initCmd_runAnyEntry(t *testing.T) {
```
Comment (henriquemoraeszup) [inline pkg/cmd/init_test.go:34]: Can you rename to CameCase?

**Review by henriquemoraeszup — APPROVED** [review]
Niiice, just a comment

**Pull request merged**